### ISSUE TITLE
Error message

### ISSUE THREAD

**Issue by ghost** [bottlepy/bottle-beaker#1]
When using your example application there's the following error:

prompt> python sample.py
Traceback (most recent call last):
  File "sample.py", line 3, in <module>
    from bottle.ext.beaker.middleware import SessionMiddleware
ImportError: No module named middleware

**Comment by robinmalburn** [comment]
I'm getting the same import error.  I'm running:

Python 2.7
bottle 0.12.5
bottle-beaker 0.1.1
Beaker 1.6.4

Bypassing the bottle-beaker plugin and using the vanilla beaker example from http://bottlepy.org/docs/dev/recipes.html#keeping-track-of-sessions works perfectly, though, so there's a quick workaround for anyone in need at least.

**Comment by avelino** [comment]
'm rewriting the plugin!

**Comment by swill** [comment]
Me too.  
On my Mac I install beaker with `pip install bottle-beaker` and everything works fine.
On my Ubuntu box, I do `pip install bottle-beaker` and I get the following error when I try to use it.

```
>>> from beaker.middleware import SessionMiddleware
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named beaker.middleware
```

It is strange because it is the same version of beaker.

On Mac:

```
$ pip show bottle-beaker
---
Metadata-Version: 1.1
Name: bottle-beaker
Version: 0.1.0
Summary: Bottle plugin beaker, WSGI middleware for sessions and caching
Home-page: https://github.com/bottlepy/bottle-beaker
Author: Thiago Avelino
Author-email: thiago@avelino.xxx
License: MIT
Location: /Library/Python/2.7/site-packages
Requires: 
```

On Ubuntu:

```
$ pip show bottle-beaker
---
Name: bottle-beaker
Version: 0.1.0
Location: /usr/local/lib/python2.7/dist-packages
Requires: 
```

Anyone know how to get around this?  

I just tested cloning the git repo and using `python setup.py install` and that also results in the same error when trying to import it on Ubuntu.  :(  Pretty confused...

**Comment by swill** [comment]
I was able to get this working by not using `bottle-beaker` and using this instead `pip install beaker`.  Not entirely sure what the deal is with this, but I am not getting the error anymore...

**Comment by etopen** [comment]
I had the same issue but solved via installing beaker with 'pip install beaker'

**Comment by avelino** [comment]
Sorry one year delay, fixed on version 0.1.2 (https://github.com/bottlepy/bottle-beaker/releases/tag/0.1.2)
